sr/robot/utils.py

```diff
@@ -199,3 +201,27 @@ def get_USB_identity(port: ListPortInfo) -> BoardIdentity:
         logger.warning(
             f"Failed to pull identifying information from serial device {port.device}")
         return BoardIdentity()
+
+
+def ensure_atexit_on_term() -> None:
```

Comment:
This implementation is identical to https://github.com/sourcebots/sbot/pull/334, just copied. I'd suggest merging the open PR upstream and rebasing the PR. This keeps the commit history cleaner and the PR more accurate.